code/__DEFINES/security.dm

```diff
@@ -0,0 +1,41 @@
+#define DETSCAN_CATEGORY_FINGERS "Prints"
+#define DETSCAN_CATEGORY_BLOOD "Blood"
+#define DETSCAN_CATEGORY_FIBER "Fibers"
+#define DETSCAN_CATEGORY_DRINK "Reagents"
+#define DETSCAN_CATEGORY_ACCESS "ID Access"
+
+// custom categories section
+#define DETSCAN_CATEGORY_NOTES "Additional Notes"
+#define DETSCAN_CATEGORY_ILLEGAL "Illegal Tech"
+#define DETSCAN_CATEGORY_SYNDIE "Syndicate Tech"
+#define DETSCAN_CATEGORY_HOLY "Holy Data"
+#define DETSCAN_CATEGORY_SETTINGS "Active Settings"
+
+/// defines the order categories are displayed, with the standard ones first, then misc, then finally the closing remarks.
+#define DETSCAN_DEFAULT_ORDER list(\
```

Comment:
```suggestion
#define DETSCAN_DEFAULT_ORDER(...) list(\
```

Nit: This should be called like `DETSCAN_DEFAULT_ORDER()`, because creating a list is a potentially costly side effect